tests/integration/integration-test.py

```diff
@@ -34,73 +38,141 @@ def setup(cls, config_path, openapi_path):
-            {'number': None, 'size': 25, 'expected_status_code': 200},
-            {'number': 999, 'size': 1, 'expected_status_code': 200},
-            {'number': -1, 'size': 25, 'expected_status_code': 400},
-            {'number': 1, 'size': -1, 'expected_status_code': 400},
-            {'number': 1, 'size': 501, 'expected_status_code': 400}
-        ]
-        nullable_fields = ['owner']
-        for pagination in testing_paginations:
-            params = {f'page[{k}]': pagination[k] for k in ['number', 'size']}
-            expected_status_code = pagination['expected_status_code']
-            resource = (
-                'PetResource' if expected_status_code == 200
-                else 'ErrorObject'
-            )
-            response = utils.test_endpoint(self, endpoint, resource,
-                                           expected_status_code,
-                                           query_params=params,
-                                           nullable_fields=nullable_fields)
-            content = utils.get_json_content(self, response)
-            if expected_status_code == 200:
-                try:
-                    meta = content['meta']
-                    num = pagination['number'] if pagination['number'] else 1
-                    size = pagination['size'] if pagination['size'] else 25
-
-                    self.assertEqual(num, meta['currentPageNumber'])
-                    self.assertEqual(size, meta['currentPageSize'])
-                except KeyError as error:
-                    self.fail(error)
-
-    # Test case: GET /pets/{id}
-    def test_get_pet_by_id(self, endpoint='/pets'):
-        valid_pet_ids = self.test_cases['valid_pet_ids']
-        invalid_pet_ids = self.test_cases['invalid_pet_ids']
-
-        for pet_id in valid_pet_ids:
-            resource = 'PetResource'
-            utils.test_endpoint(self, f'{endpoint}/{pet_id}', resource, 200)
-
-        for pet_id in invalid_pet_ids:
-            resource = 'ErrorObject'
-            utils.test_endpoint(self, f'{endpoint}/{pet_id}', resource, 404)
+                actual_state = resource['attributes']['location']['state']
+                self.assertEqual(actual_state, state)
+        for state in invalid_states:
+            params = {'filter[state]': state}
+            utils.test_endpoint(self, endpoint, 'ErrorObject', 400,
+                                query_params=params)
+
+    # Test GET parks/{id}
+    def test_get_park_by_id(self):
+        valid_park_ids = self.test_cases['valid_park_ids']
+        invalid_park_ids = self.test_cases['invalid_park_ids']
+        for park_id in valid_park_ids:
+            response = utils.test_endpoint(self, f'/parks/{park_id}', park_res,
+                                           200)
+            actual_id = response.json()['data']['id']
+            self.assertEqual(actual_id, park_id)
+        for parkId in invalid_park_ids:
```

Comment:
Oh, I missed this one. We should use lowercase_snake_case for the [naming variable and function](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) in Python. 

> Function names should be lowercase, with words separated by underscores as necessary to improve readability